compiler/src/iree/compiler/GlobalOptimization/Utils.h

```diff
@@ -7,27 +7,39 @@
 #define IREE_COMPILER_GLOBALOPTIMIZATION_UTILS_H_
 
 #include <optional>
+#include "mlir/Dialect/Utils/StaticValueUtils.h"
```

Comment:
This is not needed for header file right? 

I don't see a requirement for this after skimming through the `Utils.cpp` file. If it is needed, can you move it to `Utils.cpp`?

Reply:
My VSCode must have automatically added that. I can replace both of those includes with just `mlir/Support/LogicalResult.h`